### PR TITLE
Add an explanation for managing data persistent

### LINKED ISSUE #720: Current Dockerfile configuration is not for persistance
Related read: https://docs.docker.com/storage/#more-details-about-mount-types

Current configuration is volatile. The Dockerfile and the documentation don't provide a way to a persistent state between the node instances.

Dockerfile: https://github.com/CodeChain-io/codechain/blob/f2909594787e128a97552f982e1092e9d18c8705/docker/ubuntu/Dockerfile
Documentation: https://github.com/CodeChain-io/codechain/blob/master/README.md#setup

### PR DESCRIPTION
**Pull request by HoOngEe**
This should be merged after the #1499 is merged
It closes #720

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,23 @@ docker run -it codechain-io/codechain:branch_or_tag_name
 
 This should result in CodeChain running.
 
+#### Making local database and keys persistent
+
+CodeChain depends on the local database and keys commonly stored under the directories `keys` and `db`. A Docker container is independent of host environment and other Docker images. Therefore, in case of running a new Docker container with an image with a new CodeChain version or even with the same image, the database and keys are not persistent. To solve the problem, one can take advantage of Docker's volume option. With the command below,
```
Comment (ScarletBlue) [inline README.md]: CodeChain depends on the local database and keys commonly stored under the directories `keys` and `db`. A Docker container is independent of host environment and other Docker images. Therefore, when running a new Docker container with an image with a new CodeChain version or even with the same image, the database and keys are not persistent. To solve the problem, one can take advantage of the Docker's volume option. With the command below,

Comment (HoOngEe) [reply README.md]: Thank you

**Comment by sgkim126** [comment]
@HoOngEe Please resolve the conflicts

**Comment by HoOngEe** [comment]
I resolved the conflicts

File: codechain/config/presets/config.prod.toml [hunk 2]
```diff
@@ -3,6 +3,7 @@ quiet = false
 base_path = "."
 chain = "mainnet"
 
+
```
Comment (sgkim126) [inline codechain/config/presets/config.prod.toml]: Isn't it an unnecessary change?

Comment (HoOngEe) [reply codechain/config/presets/config.prod.toml]: Oh, I remember that I reverted it. But it has inserted in someway again!
I removed it again.

**Comment by HoOngEe** [comment]
@sgkim126 I applied your suggestion and now the commit looks clear.
